src/libcore/task/wake.rs

```diff
@@ -188,6 +188,10 @@ impl LocalWaker {
 }
 
 impl From<LocalWaker> for Waker {
+    /// Converts a `LocalWaker` into a `Waker`.
+    ///
+    /// This conversion forgets local waker and allocates a new waker with
```

Comment:
`allocate` is misleading here, since no new allocation is occurring. What about "This conversion turns a `!Sync` `LocalWaker` into a `Sync` `Waker`, allowing a wakeup object to be sent to another thread, but giving up its ability to do specialized thread-local wakeup behavior."